src/content/configuration/resolve.md

```diff
@@ -488,6 +489,42 @@ module.exports = {
 };
 ```
 
+### `resolve.importsFields`
+
+`string[]`
+
+Field names from `package.json` which are used to provide the internal requests of a package (requests starting with `#` are considered as internal).
```

Comment:
Field names? What do you mean?

Reply:
https://github.com/webpack/webpack/blob/8295202bc86b45b791c28215703f7ce4e2eac370/declarations/WebpackOptions.d.ts#L1294-L1297

Reply:
Ok it actually meant the names of the fields in the package.json to look at, got it now